packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -88,8 +88,10 @@ export function registerLocalNetwork() {
 
     rpcURLs[customL1Network.chainID] = customL1Network.rpcURL
     rpcURLs[customL2Network.chainID] = customL2Network.rpcURL
-    chainIdToDefaultL2ChainId[customL1Network.chainID] = customL2Network.chainID
-    chainIdToDefaultL2ChainId[customL2Network.chainID] = customL2Network.chainID
+    chainIdToDefaultL2ChainId[customL1Network.chainID][0] =
+      customL2Network.chainID
+    chainIdToDefaultL2ChainId[customL2Network.chainID][0] =
+      customL2Network.chainID
```

Comment:
We could do:

```suggestion
chainIdToDefaultL2ChainId[customL1Network.chainID] = [customL2Network.chainID]
chainIdToDefaultL2ChainId[customL2Network.chainID] = [customL2Network.chainID]
```

Reply:
If we have multiple L2 mapped (for example `[ChainId.Mainnet]: [ChainId.ArbitrumOne, ChainId.ArbitrumNova]`), are we fine erasing the entire array?

Reply:
This is fine, as this is only for a local network setup.